runtime/src/accounts_db.rs

```diff
@@ -7656,6 +7656,7 @@ impl AccountsDb {
                 }
             }
         }
+        // get all pubkeys in all slots
```

Comment:
"get all pubkeys in dead slots" maybe? This makes it sound like we're collecting ALL pubkeys over all existing slots